portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const URL = "https://teachablemachine.withgoogle.com/models/8A06Q2Ycp/";
+
+let model, webcam, labelContainer, maxPredictions;
+
+// load image model, set up webcam
+async function predictImage() {
+  const modelURL = URL + "model.json";
+  const metadataURL = URL + "metadata.json";
+
+  // load model, metadata
+  model = await tmImage.load(modelURL, metadataURL);
+  // Model can predict what is seen by webcam
+  // either as Squidward or Spongebob
+  maxPredictions = model.getTotalClasses();
+
+  // set up webcam
+  const flip = true; // whether to flip the webcam
+  webcam = new tmImage.Webcam(200, 200, flip);
```

Comment:
i know this was copied form the sample code, but let's make it more readable!

`webcam = new tmImage.Webcam(200, 200, flip);`
--> `webcam = new tmImage.Webcam(/* width= */ 200, /* height= */ 200, flip);`

https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments